src/sql/base/browser/ui/scrollableSplitview/scrollableSplitview.ts

```diff
@@ -316,14 +315,6 @@ export class ScrollableSplitView extends HeightMap implements IDisposable {
 		if (!types.isArray(sizes) && sizes.type === 'distribute') {
 			this.distributeViewSizes();
 		}
-
-		// Re-render the views. Set lastRenderTop and lastRenderHeight to undefined since
-		// this isn't actually scrolling up or down
-		let scrollTop = this.lastRenderTop;
-		let viewHeight = this.lastRenderHeight;
-		this.lastRenderTop = undefined;
-		this.lastRenderHeight = undefined;
-		this.render(scrollTop, viewHeight);
```

Comment:
What bug does this fix? Removing this block of code reintroduces part of the bug I fixed in #4007 where the split view wasn't getting rendered correctly when un-maximizing a grid that was previously maximized.

To reproduce, run a query with multiple result sets and maximize and then un-maximize one of them. Until you scroll enough only one of the result sets will show up, and not in the right location

Reply:
I've fixed the issue by applying a more direct fix. https://github.com/Microsoft/azuredatastudio/pull/4396/commits/b6df533c9b713728a7cedde9514d1f28fbbfdd61

Reply:
That's a much better fix than mine, thanks